packages/combobox/src/Combobox.ts

```diff
@@ -255,6 +255,7 @@ export class Combobox extends Textfield {
 
     public toggleOpen(): void {
         this.open = !this.open;
+        this.inputElement.focus();
```

Comment:
Does this cause any issues if a user clicks on a focusable element is IS NOT within the Combobox element?